csrc/torch_bindings.cpp

```diff
@@ -76,6 +76,9 @@ TORCH_LIBRARY_EXPAND(TORCH_EXTENSION_NAME, ops) {
   ops.def("advance_step", &advance_step);
   ops.impl("advance_step", torch::kCUDA, &advance_step);
```

Comment:
Do you think it's better to rename this op to `advance_step_flash_attn` to align the new FlashInfer op?

Reply:
yup, already rebased refactored, just running some quick sanity checks